diff --git a/Cargo.toml b/Cargo.toml
index f0a2e8d..0e36550 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -81,7 +81,7 @@ features = [
 version = "0.14"
 features = [
     "pem",
-    "aws_lc_rs",
+    "ring",
     "x509-parser",
 ]
 default-features = false
@@ -92,7 +92,7 @@ features = [
     "logging",
     "std",
     "tls12",
-    "aws_lc_rs",
+    "ring",
 ]
 default-features = false
@@ -114,7 +114,7 @@ features = [
 features = [
     "logging",
     "tls12",
-    "aws_lc_rs",
+    "ring",
 ]
 default-features = false
@@ -124,8 +124,7 @@ default-features = false
 [dependencies.x509-parser]
 version = "0.18"
 features = [
-    "aws-lc-rs",
+    "ring",
     "verify",
-    "verify-aws",
 ]
 default-features = false
